kubeflow/jupyter/NOTEBOOK.MD

```diff
@@ -0,0 +1,121 @@
+## Creating a Jupyter notebook in user-defined namespaces
+In order to create a notebook in a given namespace, Kubeflow expects a service account named `jupyter-notebook`, whcih allows K8s resource access,  and a secret named `user-gcp-sa,` which allows GCP service access, to exist in the namespace. 
+By default these serice acounts exist in `kubeflow` namespace so that you are able to create notebooks in this namespace. 
+You can also create jupyter notebooks in other namespaces than `kubeflow`. However, before creating any notebook you have to make sure that you create the above servic eaccounts in your desired namespace. 
+To this end, you can simply copy the aforementioned service accounts from `kubeflow` namespace to your desired namespace as follows:
+``` 
+kubectl get secret user-gcp-sa  --namespace=kubeflow --export -o yaml |\
+ kubectl apply --namespace=<your-desired-namespace> -f -
+```
+```
+kubectl get serviceaccount jupyter-notebook  --namespace=kubeflow --export -o yaml |\
+ kubectl apply --namespace=<your-desired-namespace> -f -
+``` 
+
+Alternatively, if you would like to leverage GCP services and have access to K8s resources (e.g, to make use of  Kubeflow Fairing functionality) using your account you can  manually create 
+them in your namespace as follows:
+* Create a GCP service account with your desired role (to use Kubeflow Fairing, choose `roles/editor`):
+```
+export PROJECT_ID=<your-project-id>
+export SA_NAME=<your-sa-name>
+export NAMESPACE=<your-desired-namespace>
+export GCPROLES=<your-desired-gcp-role>
+gcloud iam service-accounts create $SA_NAME
+gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member serviceAccount:$SA_NAME@$PROJECT_ID.iam.gserviceaccount.com \
+    --role $GCPROLE
+```
+* Create a key from your GCP service account:
+```
+gcloud iam service-accounts keys create <path-to-key>/key.json \
+    --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com
+```
+* Create a secret from your key JSON file and name it `user-gcp-sa`:
+
+```
+kubectl create secret generic user-gcp-sa -n $NAMESPACE\
+  --from-file=user-gcp-sa.json=<path-to-key>/key.json
+```
+* Create a service account and bind it  to a role which has permission for K8s resources in your namespace. 
```

Comment:
For MVP can we use namespace default service account?

Reply:
> For MVP can we use namespace default service account?

What is MVP?

Reply:
minimum viable product